components/customs/header.tsx

```diff
@@ -1,151 +1,192 @@
 "use client";
 
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import Link from "next/link";
-import Image from "next/image";
-import {
-  Select,
-  SelectContent,
-  SelectItem,
-  SelectTrigger,
-  SelectValue,
-} from "@/components/ui/select";
 import {
   NavigationMenu,
   NavigationMenuContent,
   NavigationMenuItem,
-  NavigationMenuLink,
   NavigationMenuList,
   NavigationMenuTrigger,
-  navigationMenuTriggerStyle,
+  NavigationMenuViewport,
 } from "@/components/ui/navigation-menu";
 import ListItem from "@/components/customs/list-item";
+
 import { ArbovirusPageSectionId } from "@/app/constants";
 
-type navMenuItem = {
+import { usePathname } from "next/navigation";
+import { cn } from "@/lib/utils";
+
+type NavMenuItem = {
   title: string;
   href: string;
   description: string;
 };
 
-const serotracker: navMenuItem[] = [
+const serotrackerNavItems: NavMenuItem[] = [
   {
     title: "Dashboard",
     href: "/pathogen/sarscov2/dashboard",
     description: "A dashboard for SarsCov2 seroprevalence data",
   },
   {
-    title: "Analysis",
-    href: "/pathogen/sarscov2/analyze",
+    title: "Data",
+    href: "/pathogen/sarscov2/dashboard",
+    description:
+      "A collection of visualizations and tabular data tools for our collection of SarsCov2 data",
```

Comment:
```suggestion
      "View or download our entire SarsCov2 dataset",
```
For consistency with the other one. The visualizations one should probably be "A collection of visualizations for our SarsCov2 dataset" as well